hoot-services/src/main/java/hoot/services/controllers/grail/GrailResource.java

```diff
@@ -656,6 +660,12 @@ public Response pullOverpassToDb(@Context HttpServletRequest request,
         InternalCommand setFolder = updateParentCommandFactory.build(jobId, folderId, layerName, user, this.getClass());
         workflow.add(setFolder);
 
+        // Clean up pulled files
+        ArrayList<File> deleteFiles = new ArrayList<>();
+        deleteFiles.add(workDir);
+        InternalCommand cleanFolders = removeFilesCommandFactory.build(jobId, deleteFiles);
+        workflow.add(cleanFolders);
+
```

Comment:
figured we could remove these files at the end of the pull?

Reply:
Yep, but the temp file output path get's cleaned up by tomcat when the service stops, so not as critical to explicitly remove these files (or the export files).